### ISSUE TITLE
Adding teams

### ISSUE THREAD

**Issue by RaphaelAlvez** [ArnoldSmith86/virtualtabletop#50]
On the player's list interface, there would be the option to input a team alongside the user name.

Teams would work as an array of players and would be used in the same place as owners.
On hands, there would be an option to separate by team.

**Comment by ArnoldSmith86** [comment]
Yeah, initially I thought the color would be used for that but I agree that it should be separate.

**Comment by robartsd** [comment]
If we add a team attribute to players, we could add a childrenPerTeam property to holders for a simple way to enable a team hand. Not sure if owner property should use team names in addition to player names or if a new team property would be more appropriate.

Perhaps both players and teams should have colors. Cursors could use one of the colors for the fill and the other as a border.
